components/web_server.rst

```diff
@@ -65,22 +71,54 @@ Configuration variables:
           auth:
             username: admin
             password: !secret web_server_password
+            
+    Example web_server configuration using version 2:
```

Comment:
This makes it sound like version 2 is not the default. It's the new and modern version and we should promote that. Why do we even still include the v1 ?

Reply:
@jesserockz  @balloob 
Hi,  Currently version 1 is the default and you get the existing behavour.  I didn't want to force V2, although perhaps this is the better choice. In breaking changes we could mention if you can select back to version 1?

I'm happy to make 2 the default if you guys are happy!